crates/bitwarden-core/src/client/internal.rs

```diff
@@ -116,6 +116,12 @@ impl InternalClient {
         *self.flags.write().expect("RwLock is not poisoned") = Flags::load_from_map(flags);
     }
 
+    #[allow(missing_docs)]
+    #[cfg(feature = "internal")]
+    pub fn set_flags(&self, flags: &Flags) {
+        *self.flags.write().expect("RwLock is not poisoned") = flags.clone();
+    }
```

Comment:
We already have `load_flags` above, and `#[allow(missing_docs)]` is not allowed on new functions.